src/System.Private.ServiceModel/src/System/ServiceModel/Security/Tokens/ISecurityContextSecurityTokenCache.cs

```diff
@@ -8,7 +8,7 @@
 
 namespace System.ServiceModel.Security.Tokens
 {
-    public interface ISecurityContextSecurityTokenCache
+    internal interface ISecurityContextSecurityTokenCache
```

Comment:
>internal [](start = 4, length = 8)

is there any risk with this change?  Is this a breaking change for existing customers?

Reply:
This is not in our contract. This was changed from public to internal as one of the types in one of the defined methods was moved to internal which a public interface isn't allowed to reference.